mip/driver_stm32.c

```diff
@@ -1,5 +1,6 @@
 #include "mip.h"
 
+#ifndef MG_ENABLE_DRIVER_IMXRT1020
 #if MG_ENABLE_MIP && \
     (!defined(MG_ENABLE_DRIVER_TM4C) || MG_ENABLE_DRIVER_TM4C == 0)
```

Comment:
See #2020 , I modded this driver to check for its own enable macro.
That macro is defined in mip.h to 1 when no other clashing driver is selected